src/main/java/study/racing/domain/RacingGame.java

```diff
@@ -1,37 +1,30 @@
 package study.racing.domain;
 
-import java.util.ArrayList;
 import java.util.List;
 
 public class RacingGame {
-    private final List<Car> cars;
-    private final RandomNumber randomNumber;
+    private final Cars cars;
+    private final RoundStatus roundStatus;
 
-    public RacingGame(int carCount, RandomNumber randomNumber) {
-        this.cars = new ArrayList<>();
-        for (int i = 0; i < carCount; i++) {
-            this.cars.add(new Car());
-        }
-        this.randomNumber = randomNumber;
+    public RacingGame(List<String> carNames, RoundCount maxRoundCount) {
```

Comment:
```suggestion
    public RacingGame(String carNames, int roundCount) {
```
이와 같은 생성자도 있다면 외부에서 RacingGame을 더 편하게 생성할 수 있지 않을까?
주 생성자도 추가한 후 주 생성자를 호출